src/test/java/de/rabea/game/GameTest.java

```diff
@@ -19,9 +19,9 @@ public void setup() {
     }
 
     @Test
-    public void playsHuman3x3GameOnce() {
-        FakeHumanPlayer fakePlayer1 = new FakeHumanPlayer(fakeConsoleUI, X);
-        FakeHumanPlayer fakePlayer2= new FakeHumanPlayer(fakeConsoleUI, O);
+    public void playsHumanGameOnce() {
+        FakeConsolePlayer fakePlayer1 = new FakeConsolePlayer(fakeConsoleUI, X);
+        FakeConsolePlayer fakePlayer2= new FakeConsolePlayer(fakeConsoleUI, O);
         fakePlayer1.futureMoves(0,1,2);
         fakePlayer2.futureMoves(4,5);
```

Comment:
Jim's comment was that these numbers (for future moves) seem random. How can I improve this? I could hide them inside a method with a more descriptive name for example?